mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
         name: 'inspectTemplateName'
         description: The name of the template to run when this job is triggered.
         required: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'inspectConfig'
+        description: The core content of the template.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'excludeInfoTypes'
+            description: When true, excludes type information of the findings.
+          - !ruby/object:Api::Type::Boolean
```

Comment:
To send the false value, send_empty_value: true is needed for this field.

Reply:
Isn't this field a regular boolean value which can be sent true/false without send_empty_value: true? I found the same field in resource `dlp_inspect_template` in a similar way [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/654a251f6174c23822d8ec8ff8d543442fee84b6/mmv1/products/dlp/InspectTemplate.yaml#L82). Similar to above declared attribute `excludeInfoTypes` .